server/test/read-time.spec.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
+import { readTime } from '../utils';
+
+
+describe('Read time checks', () => {
+  it('should successfully return read time of a content', async () => {
+    const readtime = await readTime('<span>Like seriously</span> <p>the car and the bus</p>');
+    expect(readtime).to.be.a('string');
```

Comment:
@geekdanitek Good test cases, I think we can make the test can be more comprehensive like checking for the expected read time for an article or read time for a text passed into the function.

Reply:
Okm will implement this.

Reply:
Implemented already.